.github/workflows/nodejs.yml

```
@@ -0,0 +1,78 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+# It will also deploy the build to two different repos with https://github.com/peaceiris/actions-gh-pages
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [13.3.0]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm ci
+        working-directory: ./web
+      - run: npm run check-format
+        working-directory: ./web
+      - run: npm run lint
+        working-directory: ./web
+      - run: npm run test
+        working-directory: ./web
+      - run: npm run build
+        working-directory: ./web
+    env:
+      NODE_OPTIONS: '--max-old-space-size=4096'
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Prepare imjoy.lib deploy
+        run: |
+          # Copy dirs and files and that we want to update in imjoy.lib.
+          mkdir imjoy-core && \
+          cp -Rf ./web/dist/* ./imjoy-core && \
+          rm -rf ./imjoy-core/static/* && \
+          rm -rf ./imjoy-core/docs && \
+          rm -rf ./imjoy-core/CNAME && \
+          rm -rf ./imjoy-core/index.html && \
+          rm -rf ./imjoy-core/service-worker.js && \
+          cp -Rf ./web/dist/static/icons ./imjoy-core/static/icons && \
+          cp -Rf ./web/dist/static/iconfont ./imjoy-core/static/iconfont && \
+          cp -Rf ./web/dist/static/jailed ./imjoy-core/static/jailed
+      - name: Deploy ImJoy repo
+        uses: peaceiris/actions-gh-pages@v3.5.0
+        with:
+          deploy_key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
+          publish_dir: ./web/dist
+          commit_message: |
+            ${{ format('Deploy to GitHub Pages: {0}', github.sha) }}
+      - name: Deploy imjoy.lib repo
+        uses: peaceiris/actions-gh-pages@v3.5.0
+        with:
+          deploy_key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
```

I set a different key for lib.imjoy.io
```suggestion
          deploy_key: ${{ secrets. ACTION_DEPLOY_KEY_FOR_LIB }}
```